docs/docs/workflows/terraform-provider.md

```diff
@@ -34,7 +34,10 @@ This example shows how to set up a Constellation cluster with the reference IAM
   Optionally, you can prefix the `terraform apply` command with `TF_LOG=INFO` to collect [Terraform logs](https://developer.hashicorp.com/terraform/internals/debugging) while applying the configuration. This may provide helpful output in debugging scenarios.
   </TabItem>
   <TabItem value="azure" label="Azure">
-  When creating a cluster on Azure, you need to manually patch the policy of the MAA provider before creating the Constellation cluster, as this feature isn't available in Azure's Terraform provider yet. The Constellation CLI provides a utility for patching, but you
+
```

Comment:
Instead of the triple-nested tabs here, I'd prefer to have an info block indicating that the patching is necessary for SEV-SNP only.

Then, we could simply share the text for applying for both (since it's the same for SNP and TDX).